brain/circuit.h

```diff
@@ -58,6 +58,8 @@ class Circuit
      */
     BRAIN_API explicit Circuit(const brion::BlueConfig& blueConfig);
 
+    BRAIN_API Circuit(Circuit&& other);
```

Comment:
= default doesn't work?

Reply:
Not here, because the declaration of Impl is not complete, but I can use it in the .cpp file